code/modules/mob/language.dm

```diff
@@ -410,6 +410,19 @@
 	flags = RESTRICTED | HIVEMIND
 	follow = 1
 
+
+/datum/language/terrorspider
+	name = "TerrorSpider"
+	desc = "Terror spiders have a limited ability to commune over a psychic hivemind, similar to xenomorphs."
+	speech_verb = "chitters"
+	ask_verb = "chitters"
+	exclaim_verb = "chitters"
+	colour = "alien"
+	key = "ts" // not valid, but also not used. This is their default language, so they don't need to use the key for it.
```

Comment:
@Fox-McCloud https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/say.dm#L141

it copies 3 characters then uses trim_right() to remove whitespace if present